sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -92,6 +92,11 @@ public DocTableInfo getDroppableTable(TableIdent tableIdent) {
             throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
                 "%s is an alias and hence not dropable.", tableInfo.ident()));
         }
+        if (docTableInfo.closed()) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "The table %s is closed. Write, Drop or Alter operations (other than OPEN) are not supported",
```

Comment:
These get addressed in the other PR, so I'll just remove the check from here, it is not relevant (doc table info has no closed)